app/views/preferences/index.html.haml

```diff
@@ -3,3 +3,6 @@
   - each_with_list_classes %w[account display] do |section, classes|
     %li{ class: classes }
       = link_to t("links.#{section}"), dynamic_link_helper("preferences_#{section}_path"), class: "preflink"
+  - each_with_list_classes %w[blocks] do |section, classes|
+    %li{ class: classes }
+      = link_to t("links.#{section}"), dynamic_link_helper("#{section}_path"), class: "preflink"
```

Comment:
Note to myself: go look up the notes on which section we were planning to put blocking in.